pykickstart/commands/timezone.py

```diff
@@ -420,3 +420,26 @@ def parse(self, args):
                           KickstartDeprecationWarning)
 
         return self
+
+
+class F33_Timezone(F32_Timezone):
+
+    def __init__(self, writePriority=0, *args, **kwargs):
+        F32_Timezone.__init__(self, writePriority, *args, **kwargs)
```

Comment:
What about using `super()` here?

Reply:
No other commands use super() at the moment, if we want to change this we should do a refactor globally via separate PR (provided @bcl is fine with that).

Reply:
OK.